_data/wai-evaluation-tools-list/submissions/5550e2e0-ca89-11f0-a7cf-c3cfee229cd1.json

```
@@ -0,0 +1,56 @@
+{
+  "a11yloc": "",
+  "actrules": "",
+  "assists": [
+    "Generating reports of evaluation results",
+    "Displaying information within web pages",
+    "Creating an accessibility score",
+    "Providing step-by-step evaluation guidance"
+  ],
+  "automated": [
+    "Sample of pages/screens"
+  ],
+  "comments": "",
+  "contact": "j.labudda@dock.codes",
+  "features": "WCAG accessibility checker is a free tool that performs automated WCAG 2.1 A/AA audits for websites and web applications. It provides detailed reports, clear remediation guidance, manual verification features, and API integration for CI/CD pipelines.",
+  "form_name": "submission",
+  "form_version": "1",
+  "guideline": [
+    "WCAG 2.2",
+    "WCAG 2.1",
+    "WCAG 2.0"
+  ],
+  "language": [
+    "en"
+  ],
+  "license": [
+    "Free",
+    "Limited free functionality",
+    "Subscription",
+    ""
+  ],
+  "product": [
+    "Website"
+  ],
+  "provider": "Dock",
+  "publish-permission": "on",
+  "purpose": [
+    "Automated testing"
+  ],
+  "readterms": "on",
+  "release": "2025-11-01",
+  "repository": "wai-evaluation-tools-list",
+  "submission_date": "2025-11-26T05:32:43.919Z",
+  "submission_ref": "5550e2e0-ca89-11f0-a7cf-c3cfee229cd1",
+  "technology": [
+    "HTML / XHTML",
+    "CSS"
+  ],
+  "title": "WCAG Accessibility Checker",
```

```suggestion
  "title": "Dock Accessibility Checker",
```

I suggest replacing ‘WCAG’ with ‘Dock‘
- ‘WCAG’ is not part of the tool name on the provided website.
- having the provider name at the beginning of the tool name makes it easier to distinguish between similarly named tools in the list

@josehilera Would you please ask the vendor if they accept this change?